Kernel/Term.cpp

```diff
@@ -104,6 +109,9 @@ void Term::destroyNonShared()
   }
 }
 
+bool TermList::ground() const
+{ return isTerm() && term()->ground(); }
+
```

Comment:
Should we move it, then?

Reply:
That's actually not possible because `class Term` is declared after `class TermList`, so we can't call `Term::ground` in `TermList::ground` ....

Reply:
we could ofc add an 
```
inline bool TermList::ground() const
{ return isTerm() && term()->ground(); }
```
to the header if you think it will impact performance.

Reply:
Will leave this to @MichaelRawson to decide :)

Reply:
It probably will, but I'm happy to move this in a future PR if we've decided performance so far is acceptable.